typed-builder-macro/src/util.rs

```diff
@@ -85,6 +85,17 @@ pub fn strip_raw_ident_prefix(mut name: String) -> String {
     name
 }
 
+pub fn strip_raw_ident_prefix_with_status(mut name: String) -> (String, bool) {
+    let len = name.len();
+    name = strip_raw_ident_prefix(name);
+
+    if name.len() < len {
```

Comment:
1. 
    ```rust
    (name, name.len() < len)
    ```
2. `strip_raw_ident_prefix` is simple enough that it'd be simpler and more straightforward to duplicate its code here instead of 
    * Alternatively - reverse the roles, so that `strip_raw_ident_prefix` calls `strip_raw_ident_prefix_with_status` and ditches the status.
4. A boolean output "named" `status` is not very descriptive.
5. This entire thing is actually redundant - see my comment in `setter_method_name`.

Reply:
Removed in d7e98fc